front/scripts/helpers.ts

```diff
@@ -44,13 +50,30 @@ export function makeScript<T extends ArgumentSpecs>(
   Object.entries(combinedArgumentSpecs).forEach(([key, options]) => {
     argv.option(key, options);
   });
+
   argv
     .help("h")
     .alias("h", "help")
     .parseAsync()
-    .then((args) => worker(args as InferArgs<T & { execute: boolean }>))
+    .then(async (args) => {
+      const scriptLogger = logger.child({
+        execute: args.execute,
+      });
+
+      await worker(args as InferArgs<T & { execute: boolean }>, scriptLogger);
+    })
     .catch((error) => {
       console.error("An error occurred:", error);
       process.exit(1);
     });
 }
+
+export async function runOnAllWorkspaceIds(
+  worker: (workspaceId: LightWorkspaceType) => Promise<void>
+) {
+  const workspaces = await Workspace.findAll({});
+
+  for (const workspace of workspaces) {
```

Comment:
Let's  batch a bit here no?

Reply:
Would rather not, we don't have that many workspaces and when it comes to deletion, I'd rather have a clear history in the logs.